lib/util/container-flow.js

```diff
@@ -36,11 +36,11 @@ export function containerFlow(parent, context) {
    * @returns {string}
    */
   function between(left, right) {
-    let index = -1
+    let index = context.join.length
     /** @type {ReturnType<Join>} */
     let result
 
-    while (++index < context.join.length) {
+    while (--index >= 0) {
```

Comment:
```suggestion
    while (index--) {
```

This does the same right? I think I like it better